src/object/srv_obj_migrate.c

```diff
@@ -40,6 +40,11 @@
 #include "obj_rpc.h"
 #include "obj_internal.h"
 
+/* TODO: import this srv_internal.h?
+ * Seems like pulling in that file might be a bit messy
+ */
+int ds_cont_tgt_destroy(uuid_t pool_uuid, uuid_t cont_uuid);
```

Comment:
(style)  externs should be avoided in .c files

Reply:
Would like some comments on the right thing to do here. I added this API call to srv_internal.h, like many other similar APIs. There doesn't seem to be a good way to include that header here though?